### PR TITLE
Add pipeline for unit tests

### PR DESCRIPTION
**Pull request by AramKoorn**
Hey, I added a simple ci pipeline. I noticed some requirements were missing so I added that to the setup.py file as well. I wanted to add a badge to the readme but I was not sure how to do it with a rst file.

### CODE REVIEW

**Comment by lucabasa** [comment]
Wow, first contribution! Thank you :)

File: setup.py [hunk 1]
```diff
@@ -22,6 +22,8 @@
     'numpy>=1.19.5',
     'pandas>=1.2.0',
     'scikit-learn>=0.24.1',
+    'xgboost==1.4.2',
+    'lightgbm==3.2.1'
```
Comment (lucabasa) [inline setup.py:26]: These are not really requirements of the package. I use them for the tests just to be sure they won't break but the functionalities of the package do not depend on them. 

So probably they belong to the test_require

**Comment by AramKoorn** [comment]
I moved lightgbm and xgboost to the test dependencies but then the pipeline fails due to import error

**Comment by lucabasa** [comment]
That's interesting, since they are also both packages in the requirements.txt for development, let's merge and fix it once that is set up

**Pull request merged**